src/iosrv/srv.c

```diff
@@ -321,6 +323,17 @@ dss_nvme_poll_ult(void *args)
 
 	D_ASSERT(dx->dx_main_xs);
 	while (!dss_xstream_exiting(dx)) {
+		uint64_t now = 0;
+
+		daos_gettime_coarse(&now);
+		if (dmi->dmi_last_nvme_poll_time == 0) {
+			dmi->dmi_last_nvme_poll_time = now;
+		} else if (dmi->dmi_last_nvme_poll_time +
+			   PROGRESS_MESG_INTV < now) {
+			D_DEBUG(DB_MD, "%d progress on "DF_U64"\n",
+				dmi->dmi_xs_id, now);
+			dmi->dmi_last_nvme_poll_time = now;
+		}
```

Comment:
I just don't quite understand the above logic, if last_poll_time is 2 seconds older, then it won't be updated?

TBH, I don't see the value of this patch. If a ULT is hogging CPU, above code won't be executed, so what's the point of above code?

Reply:
the last_nvme_poll_time is updated if it is older than 2 seconds.

I can see when the last progress is called, if I can not see this message in the log, then it means some ult is hogging the CPU.